test/test_browser.py

```diff
@@ -149,10 +149,17 @@ def shell_with_script(shell_file, output_file, replacement):
   create_file(output_file, shell.replace('{{{ SCRIPT }}}', replacement))
 
 
+BROWSERS_WITH_JSPI_SUPPORT = ['chrom', 'opera']
+
+
 def is_chrome():
   return EMTEST_BROWSER and 'chrom' in EMTEST_BROWSER.lower()
 
 
+def browser_supports_jspi():
+  return EMTEST_BROWSER and any(pattern in EMTEST_BROWSER.lower() for pattern in BROWSERS_WITH_JSPI_SUPPORT)
```

Comment:
Can you instead define `is_opera` and then define `browser_supports_jspi` as `is_opera() or is_chrome()`?

Reply:
Can also detect `edge` there as well, as it also uses chromium.